artifactory/commands/setup/setup.go

```diff
@@ -124,7 +128,9 @@ func (sc *SetupCommand) Run() (err error) {
 		return errorutils.CheckErrorf("unsupported package manager: %s", sc.packageManager)
 	}
 
-	if sc.repoName == "" {
+	// If the repository name is not provided, and the package manager is not Docker or Podman, prompt the user to select a repository.
```

Comment:
The comment seems redundant, as the code is self-explanatory. Why don't Docker and Podman require a repository name? Let's document that.